src/main/java/sirius/biz/storage/layer3/VirtualFile.java

```diff
@@ -247,6 +251,37 @@ public String relativePath(VirtualFile rootParent) {
         return result;
     }
 
+    /**
+     * Returns the root directory path of the current virtual file.
+     * <p>
+     * If the virtual file belongs to a known space, it is used as root, otherwise <tt>/</tt> is returned.
+     *
+     * @return the root directory path
+     */
+    @Nullable
+    public VirtualFile root() {
+        List<VirtualFile> fileList = pathList();
+        if (fileList.isEmpty()) {
+            return null;
+        }
```

Comment:
Will this ever happen? If yes, a note when null is returned should be included in the javadoc, if not, remove it :)